### ISSUE TITLE
Proton decays in w-boson. Neutron does not decay.

### ISSUE THREAD

**Issue by smtdm** [ScicraftLearn/MineLabs#281]
On Main.

Proton decays into w-boson. This should not happen.
Neutron does not decay, while it must decay into w-bosons.

**Comment by MajaVerstraeten** [comment]
Ja, dit is dwaas. Dat is geimplementeerd op basis van wat iemand zich herinnerde van Elementary craft xD 
Hier zijn de correcte decays zoals ze geimplementeerd waren in Elementary Craft

![neutrondecay](https://user-images.githubusercontent.com/69113323/184329122-79c8e433-2791-4085-b932-edcff69ddea2.png)
![weakbosondecay](https://user-images.githubusercontent.com/69113323/184329133-e7f44dce-96b0-48df-a258-1217c386d433.png)
![chargedpiondecay](https://user-images.githubusercontent.com/69113323/184329146-3cb29ab2-976c-4633-819e-2c9d841462b7.png)
![pionnuldecay](https://user-images.githubusercontent.com/69113323/184329153-3b9972ad-55a7-436b-a015-27ab18adc6a8.png)

**Comment by smtdm** [comment]
I tried some random things and it works but maybe a real programmer should look at it xp
Seel branch main_issue281
